src/org/opendatakit/briefcase/ui/export/ExportPanelForm.form

```diff
@@ -41,6 +41,15 @@
                   <text value="Clear All"/>
                 </properties>
               </component>
+              <component id="3e003" class="javax.swing.JLabel" binding="errorMsg">
+                <constraints/>
+                <properties>
```

Comment:
This new label should be not visible by default. If no errors are present, we should hide it.